peerinst/views/views.py

```diff
@@ -336,7 +336,7 @@ def welcome(request):
         return HttpResponseRedirect(reverse("student-page"))
 
     else:
-        return HttpResponseRedirect(reverse("assignment-list"))
+        return logout_view(request)
```

Comment:
This will force logout of admin accounts that are not necessarily teachers (or students).  Maybe we add an extra `elif` to check for `is_staff`?

Reply:
I just added it